### ISSUE TITLE
Suppress type check for line

### ISSUE THREAD

**Issue by 1tgr** [intellij-rust/intellij-rust#6806]
I have some code where one line is incorrectly flagged with a type check error:

```rust
use nalgebra::DVector;

fn main() {
    let a: DVector<f64> = DVector::from_vec(vec![1.0, 2.0, 3.0]);
    let b: DVector<f64> = DVector::from_vec(vec![10.0, 20.0, 30.0]);

    let c: DVector<f64> = a.component_div(&b);
    // mismatched types [E0308] expected `DVector<f64>`, found `MatrixComponentOp<f64, Dynamic, U1, Dynamic, U1>` 

    dbg!(c);
}
```

(Note that this code compiles fine, it seems that the plugin doesn't recognise that the `DVector<f64>` and `MatrixComponentOp<f64, Dynamic, U1, Dynamic, U1>` aliases point to the same type.)

I can suppress the error with a `//noinspection RsTypeCheck` comment at the top of the function. I'd prefer to apply the suppression to the `let c: DVector<f64> = ...` line only, but adding the comment above this line has no effect.

Aside from the type check error itself, should I be able to suppress a single type check error in this way?

**Comment by dima74** [comment]
Looks like it was fixed some time ago
